### PR TITLE
Fix infrastructure deployment and quick-start notebook

### PR DESCRIPTION
**Pull request by chmstimoteo**
# Description

* Fix terraform configuration files and variables
* Making quick-start instructions clearer as per customer request
* Fixing library versions dependencies
* Fixing service account IAM roles for Compute Default SA
* Adding resource labels to the terraform resources
* Add quick start multimer notebook.

# How has this been tested?

Tested with a clean deployment on a GCP project.

# Checklist

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have successfully run the E2E tests, and have included the links to the pipeline runs below
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have updated any relevant documentation to reflect my changes
- [x] I have assigned a reviewer and messaged them

### CODE REVIEW

File: env-setup/local_provisioner.tf [hunk 1]
```diff
@@ -15,16 +15,16 @@
...
+    command = <<EOT
 _REGION=${var.region}
 _PROJECT_ID=${data.google_project.project.project_id}
 _PROJECT_NUMBER=${data.google_project.project.number}
 _NETWORK_ID=${google_compute_network.network.id}
 _NETWORK_ID=$(echo $_NETWORK_ID | sed "s#$_PROJECT_ID#$_PROJECT_NUMBER#")
 _FILESTORE_IP=${google_filestore_instance.filestore_instance.networks.0.ip_addresses.0}
 _FILESHARE=${google_filestore_instance.filestore_instance.file_shares.0.name}
-_GCS_PATH=${var.gcs_dbs_path}
+_GCS_PATH=gs://${var.gcs_dbs_path}
```
Comment (leiterenato) [inline env-setup/local_provisioner.tf:27]: @chmstimoteo Could you please change the README file explaining how the user must format this GCS path?  
This is the current explanation: 

```
GCS_DBS_PATH - the path to the GCS location of the genetic databases and model parameters. Terraform will copy the databases replicating a folder structure on GCS. Terrafom will also copy model parameters to the regional bucket. The parameters should be in the <GCS_DBS_PATH>/params
```

**Review by leiterenato — COMMENTED** [review]
@chmstimoteo Thanks for your PR.
Did you test this changes?
For Terraform I recommend deleting the files generated by `terraform init` and start from scratch.

Let me know so I can approve this PR.

Thank you

File: env-setup/local_provisioner.tf (hunk 1 above)
Comment (chmstimoteo) [reply env-setup/local_provisioner.tf]: Fixed. Pls check.

**Comment by leiterenato** [comment]
LGTM

**Review by leiterenato — COMMENTED** [review]
LGTM

**Pull request merged**